Classes/Form/AbstractFormComponent.php

```diff
@@ -151,8 +137,19 @@ public function createContainer($type, $name, $label = NULL) {
 	 * @return WizardInterface
 	 */
 	public function createWizard($type, $name, $label = NULL) {
-		/** @var WizardInterface $component */
-		$className = $this->createComponentClassName($type, 'FluidTYPO3\Flux\Form\Wizard');
+		return $this->createComponent('FluidTYPO3\Flux\Form\Wizard', $type, $name, $label);
+	}
+
+	/**
```

Comment:
Suggested revision:

```php
	/**
	 * @param string $namespace
	 * @param string $type
	 * @param string $name
	 * @param string|NULL $label
	 * @return FormInterface
	 */
	public function createComponent($namespace, $type, $name, $label = NULL) {
		/** @var FormInterface $component */
		$className = $this->createComponentClassName($type, $namespace);
```

E.g. add missing type declarations in phpdoc and rename `nameSpace` to `namespace` because it's a single word.